k6/decreaseSeats.test.js

```
@@ -0,0 +1,28 @@
+
+const token = 'Bearer eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJ0ZXN0ZXIiLCJ0b2tlblZlcnNpb24iOjEsImlkIjoiYjNhODkyYTctYWJiZS00NmViLTgxZjItNmVhNDIxY2U4ZDMyIiwicm9sZSI6IkFETUlOIiwiaXNzIjoib25ldWwtdGFuZGEiLCJpYXQiOjE3NDgzNjc2OTksImV4cCI6MTc0ODM2OTQ5OX0.PnqbCywAwBP8N1E3UZElBTZ6OQuInlasMP6VR0QwuRM'; // 반드시 갱신 필요
+const flightId = '2129373a-1741-45a4-a2b0-56914adcc906';
+
+export default function () {
+    const url = `http://localhost:19091/internal/v1/flights/${flightId}/seats/decrease?requiredSeats=1`;
+
+    const headers = {
+        Authorization: token,
+    };
+
+    let res = http.put(url, null, { headers });
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'res' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)